store/src/db/sql.rs

```diff
@@ -333,6 +339,7 @@ pub fn insert_notes(
             u64_to_value(note.note_created.sender),
             u64_to_value(note.note_created.tag),
             note.merkle_path.to_bytes(),
+            note.note_created.details.to_bytes()
```

Comment:
It seems like we are writing details as an `Option` to the database. If so, I think a better approach would be to make the column nullable (as I mentioned in one of the previous comments) and then write null on `None` and the actual value of details on `Some`.